src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/DataApiNamespaceCommandSender.java

```diff
@@ -0,0 +1,39 @@
+package io.stargate.sgv2.jsonapi.api.v1.util;
+
+import static io.restassured.RestAssured.given;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.ValidatableResponse;
+import io.stargate.sgv2.jsonapi.api.v1.NamespaceResource;
+
+public class DataApiNamespaceCommandSender
+    extends DataApiCommandSenderBase<DataApiNamespaceCommandSender> {
+  protected DataApiNamespaceCommandSender(String namespace) {
+    super(namespace);
+  }
+
+  /**
+   * "Untyped" method for sending a POST command to the Data API: caller is responsible for
+   * formatting the JSON body correctly.
+   *
+   * @param jsonBody JSON body to POST
+   * @return Response validator for further assertions
+   */
+  public DataApiResponseValidator postRaw(String jsonBody) {
```

Comment:
this looks like it can move up to the base, the only difference between this is tables is the args passed to .post()

Reply:
Not quite: URL construction different. But I can extract that out, I think.

Reply:
done, created "postInternal" for sub-classes to implement.